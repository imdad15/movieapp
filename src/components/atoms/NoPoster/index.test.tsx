import React from "react";
import { render } from "@testing-library/react";
import NoPoster from ".";

describe("NoPoster tests", () => {
  it("should match snapshot", () => {
    const { container: component } = render(<NoPoster />);
    expect(component).toMatchSnapshot();
  });
});
