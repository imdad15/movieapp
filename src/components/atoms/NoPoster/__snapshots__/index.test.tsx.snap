// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoPoster tests should match snapshot 1`] = `
<div>
  <svg
    height="120"
    viewBox="0 0 24 24"
    width="120"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 0h24v24H0z"
      fill="none"
    />
    <path
      d="M2 3.993A1 1 0 0 1 2.992 3h18.016c.548 0 .992.445.992.993v16.014a1 1 0 0 1-.992.993H2.992A.993.993 0 0 1 2 20.007V3.993zM4 5v14h16V5H4zm6.622 3.415l4.879 3.252a.4.4 0 0 1 0 .666l-4.88 3.252a.4.4 0 0 1-.621-.332V8.747a.4.4 0 0 1 .622-.332z"
    />
  </svg>
</div>
`;
